WooCommerce/Classes/Authentication/Application Password/ApplicationPasswordTutorial.swift

```diff
@@ -78,9 +78,13 @@ struct ApplicationPasswordTutorial: View {
                     .frame(maxWidth: .infinity, alignment: .leading)
                     .padding([.horizontal, .top])
 
-                Image(uiImage: .appPasswordTutorialImage)
-                    .resizable()
-                    .aspectRatio(contentMode: .fit)
+                HStack(spacing: .zero) {
+                    Image(uiImage: .appPasswordTutorialImage)
+                        .resizable()
+                        .aspectRatio(contentMode: .fit)
+                        .frame(maxWidth: Layout.imageMaxWidth)
```

Comment:
One thing I forgot: we should multiply this with the scale of the device using `ScaleMetric` to make the image scalable.

Reply:
Nice catch, updated here ab5f628c86de5bc4abefac7e71261ba78869964a